fcrypt/fcrypt.go

```diff
@@ -125,15 +135,74 @@ type certificateChainInfo struct {
 
 // New create context for crypto operations.
 func New(
-	provider CertificateProvider, cryptocontext *cryptutils.CryptoContext,
+	provider CertificateProvider, cryptocontext *cryptutils.CryptoContext, cfg *config.Config,
```

Comment:
to avoid dependencies from "github.com/aoscloud/aos_communicationmanager/config" use one parameter for URL instead of full config

Reply:
Good point

Reply:
Done